sdk/keyvault/Azure.Security.KeyVault.Keys/samples/Sample2_BackupAndRestore.md

```diff
@@ -0,0 +1,56 @@
+# Back up and restore a key
+
+This sample demonstrates how to back up and restore a Key from Azure Key Vault.
+To get started, you'll need a URI to an Azure Key Vault. See the [README](../README.md) for links and instructions.
+
+## Creating a KeyClient
+
+To create a new `KeyClient` to create, get, update, or delete keys, you need the endpoint to a Key Vault and credentials.
+You can use the `DefaultAzureCredential` to try a number of common authentication methods optimized for both running as a service and development.
+
+In the sample below, you can set `keyVaultUrl` based on an environment variable, configuration setting, or any way that works for your application.
+
+```C# Snippet:KeysSample2KeyClient
+var client = new KeyClient(new Uri(keyVaultUrl), new DefaultAzureCredential());
+```
+
+## Creating a key
+
+Let's create a RSA key valid for 1 year.
+If the key already exists in the Key Vault, then a new version of the key is created.
+
+```C# Snippet:KeysSample2CreateKey
+string rsaKeyName = $"CloudRsaKey-{Guid.NewGuid()}";
+var rsaKey = new CreateRsaKeyOptions(rsaKeyName, hardwareProtected: false)
+{
+    KeySize = 2048,
+    ExpiresOn = DateTimeOffset.Now.AddYears(1)
+};
+
+KeyVaultKey storedKey = client.CreateRsaKey(rsaKey);
+```
+
+## Backing up a key
+
+Backups are good to have if in case keys get accidentally deleted by you.
```

Comment:
I don't think "Backups are good to have..." sentence adds anything.  I think users get it and "deleted by you" is a little too personal.